aws/resource_aws_docdb_cluster_instance_test.go

```diff
@@ -0,0 +1,457 @@
+package aws
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/docdb"
+)
+
+func TestAccAWSDocDBClusterInstance_importBasic(t *testing.T) {
```

Comment:
I actually missed this comment in the last PR. I'll raise a separate PR later fixing the other tests.

Reply:
I figured it was accidental and took care of it last night since it was just minor testing-related items 😄: https://github.com/terraform-providers/terraform-provider-aws/commit/c465f0eb0c5dcfdaefa33049947bb81a4380e410